terraform/modules/hub/hub_saml_soap_proxy.tf

```diff
@@ -17,27 +15,31 @@ module "saml_soap_proxy_ecs_asg" {
 }
 
 data "template_file" "saml_soap_proxy_task_def" {
-  template = "${file("${path.module}/files/tasks/stub.json")}"
+  template = "${file("${path.module}/files/tasks/hub-saml-soap-proxy.json")}"
 
   vars {
-    app = "saml_soap_proxy"
+    image_and_tag = "${local.tools_account_ecr_url_prefix}-verify-saml-soap-proxy:latest"
+    domain        = "${var.domain}"
+    deployment    = "${var.deployment}"
   }
 }
 
 module "saml_soap_proxy" {
   source = "modules/ecs_app"
 
-  deployment            = "${var.deployment}"
-  cluster               = "saml-soap-proxy"
-  domain                = "${var.domain}"
-  vpc_id                = "${aws_vpc.hub.id}"
-  lb_subnets            = ["${aws_subnet.internal.*.id}"]
-  task_definition       = "${data.template_file.saml_soap_proxy_task_def.rendered}"
-  container_name        = "stub"
-  container_port        = "8080"
-  number_of_tasks       = 1
-  health_check_protocol = "HTTP"
-  tools_account_id      = "${var.tools_account_id}"
+  deployment                 = "${var.deployment}"
+  cluster                    = "saml-soap-proxy"
+  domain                     = "${var.domain}"
+  vpc_id                     = "${aws_vpc.hub.id}"
+  lb_subnets                 = ["${aws_subnet.internal.*.id}"]
+  task_definition            = "${data.template_file.saml_soap_proxy_task_def.rendered}"
+  container_name             = "saml-soap-proxy"
+  container_port             = "8080"
+  number_of_tasks            = 1
+  health_check_protocol      = "HTTP"
+  health_check_path          = "/saml-soap-proxy/matching-service-health-check"
+  tools_account_id           = "${var.tools_account_id}"
   instance_security_group_id = "${module.saml_soap_proxy_ecs_asg.instance_sg_id}"
   certificate_arn            = "${data.aws_acm_certificate.wildcard.arn}"
+  image_name                 = "verify-saml-soap-proxy"
```

Comment:
What's the difference between `image_name` and `image_and_tag` in the variables?

Reply:
yeah we could use image_name and image_tag

and combine them in the right places

right now image_and_tag goes into the task definition

and image_name is used for permissions to pull the image